docs/platform/developing/contributing.md

```diff
@@ -33,7 +33,7 @@ To submit a patch to Gerrit:
    >
    > Valid values for `<remote_branch_name>` are:
    > - `tizen`: corresponds to the branch of the latest Tizen version
-   > - `tizen_3.0`: corresponds to the Tizen 3.0 branch
+   > - `tizen_<Version>`: corresponds to the Tizen <Version> branch
```

Comment:
This means that "tizen" branch is latest version and tizen_<Version> means specific version include 3.0, 4.0 and so all. how can I change this to more readable?

Reply:
@jhyuni, you are correct. Please check the following suggestion (you can also find a reference to the suggestion in the topic "Building All Packages Locally with GBS" in "Developing" section:

`tizen`: corresponds to the branch of the latest Tizen version
`tizen_<version>`: corresponds to branch of the specific Tizen version, including older versions.

Reply:
**FYI :** In md file, if you write a word in \<>, it is considered as a tag, as like \<strong>, or \<ul> in html.

In the original sentence there was \<Version>, and while rendering to web page, that \<Version> was considered as a kind of tag.
As a wrong tag (\<Version> is not a valid tag), the page removed it from the string, so in the web page you only could see the sentence without \<Version> : corresponds to the Tizen branch

To prevent it, you should wrap the word with \` mark as done in `tizen_<Version>`, 
or place escape character, /, before <, as I've done to this comment.

- <version>
- \<version>
- `<version>`

(I think you can compare the screenshot of my comment in editing mode and the comment as the rendered result.)
![image](https://user-images.githubusercontent.com/9958374/130009628-dac808d0-5c97-4f8b-a260-ba14d5571c7b.png)


In the md file, using \` is used, or as @sagarkedari did, changing to other sentence can be another way.

Reply:
@mijin85cho, thank you for letting know. I didn't use any tags for word "version" in my comments, just to make it consistent with previous bulleted sentence.

@jhyuni please update the text as below in the md file:

`tizen_<Version>`: corresponds to branch of the specific Tizen \<Version>, including older versions.

Reply:
We already mark this word using ` So currently This word is not regard tag and we can see word this in preview page.
Is this needed change this?

Reply:
In the original sentence, there was issue about tag in the second \<Version> and it is fixed by removing \<>.

It is also another way to fix it, so we don't need to update.


@jhyuni What @sagarkedari wanted to ask you to check was the second \<Version>

"\`tizen_\<Version>`: corresponds to the Tizen \<Version> branch" becomes below and @sagarkedari wanted to fix it.

![image](https://user-images.githubusercontent.com/9958374/130042153-06affbbc-b7a1-4ece-ad13-05bfe4d54161.png)

Reply:
@mijin85cho thank you for all clarifications. Out of 2 solutions (removing <> OR using escape character), @jhyuni has fixed the issue by removing <> for word "version" in the description, I am closing this conversation now.